### DISCUSSION TITLE
How to check Logs From Qfield

### DISCUSSION THREAD

**Discussion by ajithandro** [opengisch/QField#5508] [Announcements]
### How to check the error logs from the qfield application
Since we don't have steps to run  / debug  the application ,  is there any possible way there to check logs 

1. I'm trying to make some modifications in the Qfield , For that need check error logs 
2. How &  Where we can add print statements
3. How we can check error logs with the final output build of applications

Thanks & Regards
Ajith

**Comment by m-kuhn** [comment]
Generally it's easiest to develop on a desktop system. With the steps from the [dev docs](https://github.com/opengisch/QField/blob/master/doc/dev.md) it's possible to get a desktop build. If starting the app in a terminal session, output is visible. On android there is also the logcat command from adb.

> Since we don't have steps to run / debug the application

I'm not sure I understand. Using qtcreator it's perfectly possible to build, run and debug.

For adding more debug statements, best refer to [the qt logging documentation](https://doc.qt.io/qt-6/qtlogging.html) which is quite comprehensive.

**Comment by ajithandro** [comment]
Hello @m-kuhn ,
                          Thanks for the reply, Tried with the Qt Creator, Checked about importing the existing project process, and saw the reference that you shared [https://github.com/opengisch/QField/discussions/4095#discussioncomment-5517598](url) , 

Imported the project using **_CMakeLists.txt_** file and everything got setup and files are displayed but  facing an issue like the **_build configuration is currently disabled_**  in qt creator, Can you help me to move from this?

Thanks

**Reply by m-kuhn** [reply]
I have written a bit better documentation https://github.com/opengisch/QField/pull/5573/files can you give that a try, see if it works for you and report back any need for improvements?
